conf.py

```diff
@@ -390,7 +390,7 @@
         "name": "zookeeper",
         "versions": [
             {
-                "product": "3.8.3",
+                "product": "3.8.4",
```

Comment:
We need 3.8.3 **and** 3.8.4 please (have a look at the Supported versions sheet)